packages/composite/dbaas/exoscale/postgres.yml

```diff
@@ -0,0 +1,176 @@
+classes:
+  - ...common
+
+parameters:
+
+  pkg.appcat.composite.exoscalepostgresql:
+    defaultCompositionRef: "${facts:cloud}.exoscalepostgresqls.appcat.vshn.io"
```

Comment:
This doesn't make sense. Unlike object storage, we have a specific service that only works for exoscale. Even the kind is explicitly `ExoscalePostgreSQL` (it's not possible or sensible to use a cloudscale or AWS specific DBaaS composition for a `ExoscalePostgreSQL` for example). I see no need to have multiple compositions for this XRD, and thus this field doesn't need to be configurable.

Reply:
ah makes sense, we do not need for it another composition.